### DISCUSSION TITLE
how to preform scenicplus base on scATAC data and scRNA data with different barcode

### DISCUSSION THREAD

**Discussion by WeiFeng0726** [aertslab/scenicplus#305] [Q&A] [answered]
Hello, scenicplus team.

I'd like to thank you for development this wonderful tools

I notice that scATAC and scRNA data with same barode(cell label)  is necessary when i preform tutorial: 10x multiome pbmc.
however,  i preform sc-ATAC and sc-RNA seq on same sample respectively, so they have differen cell barcode.
In this situation, how can I use scenicplus.

**Comment by WeiFeng0726** [comment, marked as answer]
I'm sorry about this discussion, The solution can be found in Tutorial 3: Mix of melanoma cell lines

from scenicplus.scenicplus_class import create_SCENICPLUS_object
import numpy as np
scplus_obj = create_SCENICPLUS_object(
        GEX_anndata = adata,
        cisTopic_obj = cistopic_obj,
        menr = menr,
        multi_ome_mode = False,
        key_to_group_by = 'MMline',
        nr_cells_per_metacells = 5)

**Reply by MVemon** [reply]
Hello,

I was wondering if you were able to produce the demuxlet files from the tutorial?

**Reply by SeppeDeWinter** [reply]
Hi @marck1198 

To be clear, the tutorial does not show how to produce these files. 
These files were needed for the tutorial because we mixed several MM lines (from different human donors) in a single sequencing run. To see which cell line is which, we used demuxlet to demultiplex the data based on donor specific SNPs.

If you don't have a mix of genotypes in your data, you don't need these files at all.

All the best,

Seppe
